### PR TITLE
Zero to Hero Tutorial

### PR DESCRIPTION
**Pull request by 0xLucca**
This PR reorganizes existing tutorials into a new structure

### CODE REVIEW

**Comment by dawnkelly09** [comment]
Just as a heads up for anyone looking at this early version. I will have some commits coming to make a single page zero to hero pure tutorial with links to these guides for users to get more info. Stay tuned!

File: tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md:1]:
```suggestion
```

Comment (dawnkelly09) [reply tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md:1]: This blank line at the top is keeping the description from displaying on the cards on the index page

File: tutorials/polkadot-sdk/parachains/zero-to-hero/index.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/index.md:12]: Please add a "!!! tip" admonition with text:

Start with the "Set Up a Template" guide to get the most from this section. As you complete each guide, look for "Where to Go Next" to move to the following guide in the series.

File: tutorials/polkadot-sdk/parachains/zero-to-hero/set-up-a-template.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/set-up-a-template.md:163]:
```suggestion
    [:octicons-arrow-right-24: Build A Custom Pallet](/tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet/)
```

Comment (dawnkelly09) [reply tutorials/polkadot-sdk/parachains/zero-to-hero/set-up-a-template.md:163]: I don't feel like Reference makes sense here.

File: tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md:111]:
```suggestion
    [:octicons-arrow-right-24: Execute a Runtime Upgrade](/tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade/)
```

File: develop/parachains/maintenance/runtime-upgrades.md
Comment (dawnkelly09) [inline develop/parachains/maintenance/runtime-upgrades.md:84]:
```suggestion
    [:octicons-arrow-right-24: Add Pallets to the Runtime](/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/)
```

Comment (dawnkelly09) [reply develop/parachains/maintenance/runtime-upgrades.md:84]: What is this TODO! here?

File: tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md:263]:
```suggestion
    [:octicons-arrow-right-24: Pallet Unit Testing](/tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing/)
```

File: tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md:204]:
```suggestion
    [:octicons-arrow-right-24: Obtain Coretime](/tutorials/polkadot-sdk/parachains/zero-to-hero/obtain-coretime/)
```

File: tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md:172]:
```suggestion
    [:octicons-arrow-right-24: Add Pallets to the Runtime](/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/)
```

**Review by dawnkelly09 — CHANGES_REQUESTED** [review]
Still seeing some TODO items in here

File: tutorials/polkadot-sdk/parachains/zero-to-hero/index.md
Comment (nhussein11) [reply tutorials/polkadot-sdk/parachains/zero-to-hero/index.md:12]: fixed on [22c46b0](https://github.com/polkadot-developers/polkadot-docs/pull/254/commits/22c46b0cb19fc1521d8f2f80be1b050e20326b02)

**Review by nhussein11 — APPROVED** [review]
LGTM!

File: tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md:111]: This link and the link text are for a runtime upgrade but the title says Deploy on Paseo TestNet.

I don't see the Execute a Runtime Upgrade in the left nav but I can see the page exists if I type in the URL myself. Please fix this card to match where it should go and update the navigation to add the runtime upgrade page.

File: tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md:117]: What goes here?

Comment (dawnkelly09) [reply tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md:117]: Please check the TODO items for this page

File: tutorials/polkadot-sdk/parachains/zero-to-hero/obtain-coretime.md
Comment (dawnkelly09) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/obtain-coretime.md:112]: Please add a Where to Go Next. I know this is the last article in the series, but what should they read or try next?

**Review by dawnkelly09 — COMMENTED** [review]
Still some TODO items on the Runtime Upgrade page from Zero to Hero

**Comment by 0xLucca** [comment]
The only thing missing for this PR to be ready to go is to modify the index file of the `tutorials > polkadot-sdk > parachains` section

File: tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime.md:71]:
```suggestion
## Run Your Chain Locally
```

File: tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md:18]: We can't assume users are following these tutorials sequentially, so please link to previous tutorial.

File: tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md:10]: Can't make this assumption. Please link to previous tutorials

File: tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/deploy-to-testnet.md:204]: Use a different CTA than the title

File: tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md:1]: This tutorial doesn't show up in the left navigation. Need to update the .pages file

File: tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md:10]: Same comment here about the previous tutorials situation. Please check all of the guides and make sure we're not assuming that the user knows what the previous tutorials are

File: tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md
Comment (eshaben) [reply tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md:1]: If we don't plan on having this for launch, we need to delete it because it will show up in the search bar and on search engines. Leaving it out of the .pages file isn't enough

File: tutorials/polkadot-sdk/parachains/zero-to-hero/index.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/index.md:14]:
```suggestion
    Start with the Set Up a Template guide to get the most from this section. As you complete each guide, look for Where to Go Next to move to the following guide in the series.
```

File: tutorials/polkadot-sdk/parachains/zero-to-hero/index.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/index.md:2]:
```suggestion
title: Zero To Hero Parachain Tutorial Series
```

File: tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md:10]:
```suggestion
You have learned how to create a custom pallet in the [Build the Pallet](/tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet/){target=\_blank} tutorial; now you will see how to test the pallet to ensure that it works as expected. As stated in the [Pallet Testing](/develop/parachains/customize-parachain/pallet-testing/){target=\_blank} article, unit testing is crucial for ensuring the reliability and correctness of pallets in Polkadot SDK-based blockchains. Comprehensive testing helps validate pallet functionality, prevent potential bugs, and maintain the integrity of your blockchain logic.
```

File: tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md:172]: Don't repeat the title as the CTA. We can use something generic here like "Get started"

File: tutorials/polkadot-sdk/parachains/zero-to-hero/set-up-a-template.md
Comment (eshaben) [inline tutorials/polkadot-sdk/parachains/zero-to-hero/set-up-a-template.md:163]: Same CTA as title here

**Review by eshaben — COMMENTED** [review]
`tutorials/polkadot-sdk/parachains/index.md` has a bunch of the old links that need to be removed. I know this was already noted, but adding it to the PR so we do not forget about it before merging

File: tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md
Comment (0xLucca) [reply tutorials/polkadot-sdk/parachains/zero-to-hero/execute-runtime-upgrade.md:1]: That tutorial is blocked until a new release of the omni node fixes an issue. I'll remove it

File: develop/parachains/maintenance/runtime-upgrades.md
Comment (0xLucca) [reply develop/parachains/maintenance/runtime-upgrades.md:82]: This tutorial will not be there at launch

File: tutorials/index.md
Comment (nhussein11) [reply tutorials/index.md]: done in [a968cce](https://github.com/polkadot-developers/polkadot-docs/pull/254/commits/a968cce41219d1dcaaa8583eb10b046a1146a99a). WDYT @eshaben ?

File: tutorials/index.md
Comment (nhussein11) [reply tutorials/index.md:32]: agree! What if we put this content into an admonition element? Like info or warning?

**Review by dawnkelly09 — COMMENTED** [review]
I made a few updates to links, etc so I'll defer final signoff to @eshaben. Looks good though!

**Review by eshaben — APPROVED** [review]
LGTM!

One thing to note: the `/tutorials/polkadot-sdk/parachains/index.md` file feels like it's the index page for the Zero to Hero section. I think we should modify this page a bit after launch.

**Review by dawnkelly09 — APPROVED** [review]
🚀

**Pull request merged**
